File: solutions/julia/raindrops/1/raindrops.jl
function raindrops(number)
    raindrops = ""
    if number % 3 == 0
        raindrops = string(raindrops, "Pling")
    end
    if number % 5 == 0
        raindrops = string(raindrops, "Plang")
    end
    if number % 7 == 0
        raindrops = string(raindrops, "Plong")
    end
    if length(raindrops) == 0
        return string(number)
    end
    return raindrops
end

File: solutions/cpp/raindrops/1/raindrops.cpp
#include "raindrops.h"

namespace raindrops {
    std::string convert(size_t num) {
        if(num % 3 && num % 5 && num % 7)
            return std::to_string(num);
        std::string raindrop;
        if(num % 3 == 0)
            raindrop += "Pling";
        if(num % 5 == 0)
            raindrop += "Plang";
        if(num % 7 == 0) 
            raindrop += "Plong";
        return raindrop;
    }
}  // namespace raindrops
